src/mono/CMakeLists.txt

```diff
@@ -543,6 +539,9 @@ if (GC_SUSPEND STREQUAL "coop")
   set(ENABLE_COOP_SUSPEND 1)
 elseif(GC_SUSPEND STREQUAL "hybrid")
   set(ENABLE_HYBRID_SUSPEND 1)
+elseif(GC_SUSPEND STREQUAL "preemptive")
+else()
+  message(FATAL_ERROR "GC_SUSPEND (set to '${GC_SUSPEND}') must be one of coop, hybrid or preemptive")
```

Comment:
If you're removing the option of a default in the main CMake file, it should be removed from [here](https://github.com/dotnet/runtime/blob/master/src/mono/cmake/options.cmake#L70) as well.

Reply:
@CoffeeFlux I kind of like that it's set in options.cmake to an invalid value.  That way it's obvious if mono.proj is missing a case.  But I guess we could change the string to anything - maybe `"change in mono.proj"`.

Reply:
I think there should still be a reasonable default so this can be used without using mono.proj.

Reply:
ok, added back some code to pick a default (hybrid, unless you're on windows or watch) if GC_SUSPEND isn't set from the outside